
Loading data...
Fitting model on all data including 2023 and returning
Traceback (most recent call last):
  File "/Users/kelseydoerksen/code/landslide-forecast-nepal/run_classifier.py", line 736, in <module>
    run_rf(X_train, y_train, X_test, y_test, results, experiment, model, test_y)
  File "/Users/kelseydoerksen/code/landslide-forecast-nepal/run_classifier.py", line 394, in run_rf
    forest.fit(Xtrain, ytrain)
  File "/Users/kelseydoerksen/opt/anaconda3/envs/aq/lib/python3.10/site-packages/sklearn/ensemble/_forest.py", line 346, in fit
    X, y = self._validate_data(
  File "/Users/kelseydoerksen/opt/anaconda3/envs/aq/lib/python3.10/site-packages/sklearn/base.py", line 554, in _validate_data
    X, y = check_X_y(X, y, **check_params)
  File "/Users/kelseydoerksen/opt/anaconda3/envs/aq/lib/python3.10/site-packages/sklearn/utils/validation.py", line 1104, in check_X_y
    X = check_array(
  File "/Users/kelseydoerksen/opt/anaconda3/envs/aq/lib/python3.10/site-packages/sklearn/utils/validation.py", line 877, in check_array
    array = _asarray_with_order(array, order=order, dtype=dtype, xp=xp)
  File "/Users/kelseydoerksen/opt/anaconda3/envs/aq/lib/python3.10/site-packages/sklearn/utils/_array_api.py", line 185, in _asarray_with_order
    array = numpy.asarray(array, order=order, dtype=dtype)
  File "/Users/kelseydoerksen/opt/anaconda3/envs/aq/lib/python3.10/site-packages/pandas/core/generic.py", line 2070, in __array__
    return np.asarray(self._values, dtype=dtype)
ValueError: could not convert string to float: '2018-09-09'